plugins/cache.rst

```diff
@@ -33,22 +33,38 @@ control headers from the server as specified in :rfc:`7234`. It needs a
         [$cachePlugin]
     );
 
+.. note::
+
+    Since v1.3.0 does the ``CachePlugin`` have 2 factory methods to easily setup the plugin by caching type. See the
+    example below.
```

Comment:
i'd suggest to move most of this out of the node and instead say just that there are the factory methods for the client resp. server mode. then have the code block, and then below a note just saying "The two factory methods have been added in version 1.3.0."